commands_to_load/Mod.py

```diff
@@ -82,5 +83,398 @@ def __init__(self, bot):
+            logger.info('[Mod makechannel()] unathorized command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+
+        # Verify args
+        if not name: 
+            eObj = await em(ctx, description='Missing arguments', footer='Error in makechannel command')
+            if eObj is not False:
+                await ctx.send(embed=eObj)
+            return
+
+        # Get the MUTED role
+        MUTED_ROLE = discord.utils.get(ctx.guild.roles, name='Muted')
+
+        overwrite = {
+            ctx.guild.default_role : discord.PermissionOverwrite(mention_everyone=False),
+            MUTED_ROLE : discord.PermissionOverwrite()
+        }
+        
+        # Set the Muted roles permissions
+        setattr(overwrite[MUTED_ROLE], 'send_messages', False)
+        setattr(overwrite[MUTED_ROLE], 'manage_messages', False)
+        setattr(overwrite[MUTED_ROLE], 'manage_channels', False)
+        setattr(overwrite[MUTED_ROLE], 'manage_guild', False)
+        setattr(overwrite[MUTED_ROLE], 'manage_nicknames', False)
+        setattr(overwrite[MUTED_ROLE], 'manage_roles', False)
+
+        logger.info('[Mod makechannel()] channel permissions created')
+
+        # Check if making secret channel then append those perms
+        if secret: 
+            setattr(overwrite[ctx.guild.default_role], 'read_messages', False)
+            setattr(overwrite[ctx.guild.default_role], 'manage_messages', False)
+            setattr(overwrite[ctx.guild.default_role], 'mention_everyone', True)
+            logger.info('[Mod makechannel()] making a hidden channel')
+
+        # Create channel
+        ch = await ctx.guild.create_text_channel(name, overwrites=overwrite)
+        logger.info('[Mod makechannel()] channel "' + name + '" made by ' + str(ctx.author))
+
+        # Send message to council about channel made 
+        council = discord.utils.get(ctx.guild.channels, name="council")
+        eObj = await em(ctx, description=str(ctx.author) + ' made channel: `' + name + '`', footer='Moderator action')
+        if eObj is not False:
+            await council.send(embed=eObj)
+
+    @commands.command()
+    async def clear(self, ctx, numOfMsgs = 10.0):
+        # Deletes the last X (10 by default) msg's from channel
+        # Limits: Max 100 messages at a time AND no messages older than 2 weeks (14 days)
+
+        logger.info('[Mod clear()] clear function detected by user' + str(ctx.message.author))
+        await ctx.message.delete()
+        logger.info('[Mod clear()] invoking command deleted')
+        
+        if not ctx.message.author in discord.utils.get(ctx.guild.roles, name="Minions").members:
+            logger.info('[Mod clear()] unathorized command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+
+        numOfMsgs = int(numOfMsgs)
+
+        # Verify args
+        if numOfMsgs > 100 or numOfMsgs < 1: 

```

Comment:
you ought to do a return at the end of this `if` statement, otherwise, it'll still continue despite being given a bad range